tests/devapps/NetCoreTestApp/Program.cs

```diff
@@ -241,6 +244,33 @@ 1. IWA
 
                             break;
 
+                        case 11: // AcquireTokenForClient with multiple threads
```

Comment:
>AcquireTokenForClient with multiple threads [](http://example.com/codeflow?start=36&length=43)

did this test / sample fail before the fix?

Reply:
Yes, it fails with the code without the fix.